apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-credit/footerCreditChoices.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable wpcalypso/import-docblock */
+/* global fullSiteEditing */
+/**
+ * WordPress dependencies
+ */
+import { Icon } from '@wordpress/components';
+
+export const { footerCreditOptions, defaultCreditOption } = fullSiteEditing;
```

Comment:
Missed this at a first read: I'm questioning the value of this export (on top of the fact that I'd avoid multiple unrelated exports in the same file).

`fullSiteEditing` is a global available everywhere, so this is basically only a shortcut for a variable destructuring.
On top of this, `defaultCreditOption` is not even used in this file.

I think we could drop this and use a simpler `fullSiteEditing.footerCreditOptions` later in this file, and the destructuring in the `edit.js`.

Reply:
I think that's a fair point. Previously, I was defining the option here, so this is what it evolved to after I started declaring them from PHP. I'll change it